examples/jsm/renderers/webgl/WebGLBackend.js

```diff
@@ -47,6 +47,7 @@ class WebGLBackend extends Backend {
 		this.discard = false;
 
 		this.extensions.get( 'EXT_color_buffer_float' );
+		this.parallel = this.extensions.get( 'KHR_parallel_shader_compile' );
```

Comment:
I don't really like that we are introducing a new property just to get a constant... Maybe at least mark it private?